Adding new fields for ls cloud

babhalodia opened the pull request:
Adding new fields and fields-alias for cloud.

Commit:
Adding new fields and fields-alias for cloud.

Commit:
resolving conflict

Commit:
Adding eval for OS field

Commit:
Changing Fields alias to resolve conflict b/w userdomain & assetdomain

Commit:
Updating dashboard of Lansweeper App to make it compatible with Onprem Logs

VatsalJagani commented on lansweeper_app_for_splunk/default/data/ui/views/ls_assets.xml:
```diff
@@ -62,14 +77,19 @@
+          <query>`ls_assets_data`  Statename=$tkn_asset_state|s$  
+| eval id=coalesce(AssetID,id) 
+| dedup id 
+| rename AssetName as Name, AssetTypename as Type, Statename as State, Userdomain as UserDomain, AssetGroup as GroupName, IPAddress as IPAddress, Mac as MacAddress, OScode as OSVersion, Username as User, Firstseen as FirstSeen, Lastseen as LastSeen, LastChanged as LastChanged 
```
Let's add OS (coalesced one from SQL, for cloud, we'll coalesce from whatever we get) field after Asset Type field to see if it adds any more value to it.

babhalodia replied:
Added field OS

VatsalJagani commented on lansweeper_app_for_splunk/default/data/ui/views/ls_assets.xml:
```diff
@@ -1,7 +1,7 @@
+  <label>Assets Clone</label>
   <fieldset submitButton="false">
-    <input type="dropdown" token="tkn_site">
+    <input type="dropdown" token="tkn_site" depends="$hide$">
```
Let's remove it completely, as we are already removing its usage. If we need it in the future we can add it again.

babhalodia replied:
Removed Input

VatsalJagani commented on lansweeper_app_for_splunk/default/data/ui/views/ls_assets.xml:
```diff
@@ -31,19 +31,34 @@
       </table>
     </panel>
+    <panel>
+      <title>Assets With Lansweeper Agent</title>
```
This seems really useful. One suggestion though (because label going is bit complex, just to simplify it.):
- Let's rename it to "Lansweeper Agent", and rename the count column to AssetCount.

babhalodia replied:
Actually, I was not also satisfied but I was not able to think something else :). Made suggested changes.

Commit:
Minor Changes in Dashboard

babhalodia committed:
Update ls_assets.xml

Minor change: Change field name for Userdomain to make compatible with Cyences Dashboard name.

babhalodia committed:
Update props.conf

Minor Bug: Changing field name and making compatible with on-Prem.

babhalodia committed:
Update props.conf

babhalodia committed:
Update props.conf

Adding coalesce for Site for LS On-prem

babhalodia committed:
Update ta_lansweeper_api.py

babhalodia committed:
Update props.conf
